tests/sentry/grouping/fingerprint_inputs/fingerprint-exception-type-and-sdk-mismatch.json

```diff
@@ -0,0 +1,36 @@
+{
+  "_fingerprinting_rules": [
+    {
+      "matchers": [
+        ["sdk", "sentry.javascript.nextjs"],
```

Comment:
I understand this PR now.
You're giving support for using `sdk` as a matcher + you're adding fingerprint rules to the events as if the customer had defined them.